Gemfile.lock

```diff
@@ -7,11 +7,12 @@ GIT
 
 GIT
   remote: https://github.com/CocoaPods/Core.git
-  revision: b84eb6ab23a3e65a0ab79487fd3fe44c590cb29a
+  revision: 04d111d9deb3e09563933ae547c7c6f1a104b6a4
   branch: master
   specs:
-    cocoapods-core (1.9.0.beta.3)
+    cocoapods-core (1.9.0)
```

Comment:
this is weird as 1-9-stable is already merged into master https://github.com/CocoaPods/CocoaPods/blob/master/Gemfile.lock#L13

Reply:
is your branch properly rebased against latest master?

Reply:
still probably need to bump the SHA of cocoapods-core to ensure your other PR is available to you with the DSL.

Reply:
Sorry @dnkoutso, I have made the merge again, this branch it's outdated